rts/lra/test/basic/pom.xml

```diff
@@ -12,6 +12,7 @@
   <description>LRA basic integration tests</description>
 
   <properties>
+    <config.to.replace>standalone-microprofile.xml</config.to.replace>
```

Comment:
Are you sure this will override the value being set in the parent pom?

Reply:
The LRA axis was passing still I think so yes. In particular this change is to ensure that WildFly is ran with OpenAPI which is important for the test I am proposing to add. I will add a comment.

Reply:
ok